src/main/java/org/syncninja/service/CommitTreeService.java

```diff
@@ -33,11 +35,12 @@ public void addFileToCommitTree(String mainDirectoryPath, List<String> listOfFil
             throw new Exception(ResourceMessagingService.getMessage(ResourceBundleEnum.DIRECTORY_NOT_INITIALIZED, new Object[]{mainDirectoryPath}));
         }
         List<StatusFileDTO> untrackedFiles = fileTrackingState.getUntracked();
-        addFilesToCommitTree(untrackedFiles, mainDirectoryPath, listOfFilesToBeAdded);
+        Map<String,FileStatusEnum> directoriesState = fileTrackingState.getDirectoriesState();
+        addFilesToCommitTree(untrackedFiles, mainDirectoryPath, listOfFilesToBeAdded , directoriesState);
```

Comment:
can you please fix a space here `listOfFilesToBeAdded , directoriesState` to `listOfFilesToBeAdded, directoriesState`